packages/www/components/StreamDetails/MultistreamTargetsTable/SaveTargetDialog.tsx

```diff
@@ -233,7 +233,7 @@ const SaveTargetDialog = ({
   return (
     <AlertDialog open={isOpen} onOpenChange={onOpenChange}>
       <AlertDialogContent
-        css={{ maxWidth: 450, px: "$5", pt: "$4", pb: "$4" }}
+        css={{ maxWidth: 450, px: "$5", pt: "$4", pb: "$4", zIndex: 1 }}
```

Comment:
This is not needed, it was an unrelated change that got merged in another PR from @0xcadams

Reply:
Reverted in 614e408f7c7a468b4d321b6076a3d9ad1edfde45